R2API/ItemAPI.cs

```diff
@@ -112,12 +113,23 @@ public static class ItemAPI {
                 R2API.Logger.LogError($"Too late ! Tried to add item: {item.ItemDef.nameToken} after the item list was created");
             }
 
-            if (item.ItemDef == null) {
+            if (!item.ItemDef) {
                 R2API.Logger.LogError("Your ItemDef is null ! Can't add your item.");

```

Comment:
I know you are continuing the trend, but we should not address a 'you' when it might likely be an enduser who also sees this message